server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationAllocationIT.java

```diff
@@ -193,7 +193,7 @@ public void testAllocationWithDisruption() throws Exception {
 
     /**
      * Utility method which ensures cluster has balanced primary shard distribution across a single index.
-     * @throws Exception
+     * @throws Exception exception
```

Comment:
Just an FYI, if you've been getting javadoc warnings/failures in the tests, it might be due to building with JDK 14. The javadoc rules are supposed to be ignored for tests, but JDK-14 doesn't seem to do the right thing, though it has been end-of-life for quite awhile now.

Reply:
Ahh. Yep. I think since we have a new setting for `FsRepository`, an entry makes sense. 
@samuel-oci You can follow the `CHANGELOG` guide here - https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#changelog
Please add an entry into the `Unreleased 2.x` section since this will be backported to `2.x`